jsonexample/lib/tab_pages.dart

```diff
@@ -43,20 +41,22 @@ class BasicsPage extends StatelessWidget {
     final boldStyle =
         localTheme.bodyText2.copyWith(fontWeight: FontWeight.w600);
 
-    final dynamic dynamicListOfInts = json.decode(JsonStrings.listOfInts);
+    final dynamicListOfInts = json.decode(JsonStrings.listOfInts) as Iterable;
```

Comment:
nit: `as Iterable` works, but `json.decode` will always return Lists, right?

Reply:
True.

I was using Iterable because that's what `list.from` has in its signature. Now that I'm looking with fresh eyes, though, List makes more sense and matches the variable names better, so I've updated the code.